modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/model/DDMFormInstanceSettings.java

```diff
@@ -121,6 +122,15 @@
 	)
 	public String emailToAddress();
 
+	@DDMFormField(
+		label = "%select-object",
+		properties = {
+			"dataSourceType=data-provider", "ddmDataProviderInstanceId=objects"
+		},
+		type = "select"
+	)
+	public String objectId();
```

Comment:
@gabrielwas, why are you naming `objectId` instead of just `object`? Can you please give me more details?

Reply:
Because there I'm storing the objectId. Do you remember I had this issue regarding the types (string or long). It should be a long. I'll try to figure it out today, though.

Reply:
@gabrielwas, look what I've found: https://github.com/liferay/liferay-portal/blob/03e0dfa973554bd6458f71080122dcae3d2315e8/modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/DDMDataProviderInvokerImpl.java#L167-L185. Maybe, it can be the explanation. :wink: